### ISSUE TITLE
Empty world

### ISSUE THREAD

**Issue by Azmodan17** [dalaranwow/dalaran-wow#5532]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Started a quest "Army of the Damned" in Icecrown when I was just questing around, then logged off in the middle of the quest, logged back on and was no longer Arthas and could not complete the quest. All the soldiers were still there but nothing else, the world was empty. Flew all the way to Dalaraan and it was empty, tried dropping the quest but changed nothing. I am currently stuck in an empty world and need to be phased back in somehow.
**Expected behaviour**: Tell us what should happen instead
Should have logged back in at Arthas since I was still in the same spot for the quest.
**Steps to reproduce the problem**:

1. take quest
2. kill some ally and raise minions
3. log out and back in, should no longer be arthas and can't complete

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
http://wowwiki.wikia.com/wiki/Quest:Army_of_the_Damned
**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Jukkapekka** [comment]
Hi, 
I've got the same issue, though it's been after I abandonned the quest.
No npc's, no players, no mailboxes, no portals, etc. 
Any idea/solution?

Cheers

**Comment by Azmodan17** [comment]
Sorry I can't help, seems like something in the coding bugged out, been
waiting for a GM to put me back in the right "phase" of the world for lack
of better idea

**Comment by Golganeth** [comment]
Make a relog or kill your character once you have abandoned the quest that messes up with your phasing.
